src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -39,10 +43,25 @@ def target_project_base_name
   def validate_repositories
     return if step_instructions[:repositories].all? { |repository| repository.keys.sort == REQUIRED_REPOSITORY_KEYS }
 
+    # FIXME: This is only to help users migrate their configure_repositories steps when we introduced this breaking change. Remove this after March 1st, 2022.
+    if step_instructions[:repositories].any? { |repository| !repository.key?(:paths) }
+      errors.add(:base,
+                 "configure_repositories step: Repository paths are now set under the 'paths' key. Refer to " \
+                 'https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html#id-1.5.8.5.2.8.9.7 for an example')
```

Comment:
I would not use the ID anchor. It keeps changing...

Reply:
Somehow, certain headings have an ID, others have a string. I'll see what I can do to always have a string.

Reply:
I've updated the section IDs in this PR and the [obs-docu PR](https://github.com/openSUSE/obs-docu/pull/218). Please have another look.